spec/docs/reference-implementations.md

```
@@ -34,6 +37,26 @@ docker run -p 8085:8085 \
+### Via pipx (LangChain-based Interpreter)
+
+Install the AFM CLI using pipx:
+
+```bash
+pipx install afm-cli
+```
+
+!!! Note
+    `pipx` installs CLI tools in isolated environments and is the recommended approach on modern systems (Ubuntu 23.04+, Debian 12+, macOS with Homebrew Python, etc.), where `pip install` to the system Python may be blocked. See [pipx.pypa.io](https://pipx.pypa.io/) for installation instructions.
+
+    If you are working inside an activated virtual environment, `pip install afm-cli` works as well.
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)